proxy/http2/Http2ClientSession.cc

```diff
@@ -175,12 +175,6 @@ Http2ClientSession::start()
   read_vio  = this->do_io_read(this, INT64_MAX, this->read_buffer);
   write_vio = this->do_io_write(this, INT64_MAX, this->sm_writer);
 
-  // 3.5 HTTP/2 Connection Preface. Upon establishment of a TCP connection and
-  // determination that HTTP/2 will be used by both peers, each endpoint MUST
-  // send a connection preface as a final confirmation ...
-  // this->write_buffer->write(HTTP2_CONNECTION_PREFACE,
-  // HTTP2_CONNECTION_PREFACE_LEN);
```

Comment:
It looks like this comment comes from old draft of HTTP/2.

RFC7540 says below about connection preface from server side.

> The server connection preface consists of a potentially empty
   SETTINGS frame (Section 6.5) that MUST be the first frame the server
   sends in the HTTP/2 connection.